backend/routes/external.js

```
@@ -0,0 +1,745 @@
+      extrasMap.set(item.livreur_id, parseInt(item.nombre_extras));
+    });
+
+    // Traitement des détails par livreur (même logique que l'endpoint daily)
+    const processedDetails = details.map(livreur => {
+      // Traitement des points de vente MATA
+      let mataParPointVente = [];
+      if (livreur.mata_details_points_vente) {
+        const pointsVenteMap = new Map();
+        livreur.mata_details_points_vente.forEach(item => {
+          if (item && item.point_de_vente) {
+            const existingCount = pointsVenteMap.get(item.point_de_vente) || 0;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)